sdk/mobilenetwork/Azure.ResourceManager.MobileNetwork/src/Generated/Models/UeInfo5G.cs

```diff
@@ -0,0 +1,196 @@
+        {
+            Supi = supi;
+            Pei = pei;
+            SessionInfo = sessionInfo;
+            PerUeTnla = perUeTnla;
+            AmfUeNgapId = amfUeNgapId;
+            RanUeNgapId = ranUeNgapId;
+            LastVisitedTai = lastVisitedTai;
+            AllowedNssai = allowedNssai;
+            UeState = ueState;
+            RrcEstablishmentCause = rrcEstablishmentCause;
+            UeUsageSetting = ueUsageSetting;
+            LastActivityOn = lastActivityOn;
+            NgeNbId = ngeNbId;
+            ENbId = eNbId;
+            N3IwfId = n3IwfId;
+            WagfId = wagfId;
+            TngfId = tngfId;
+            Nid = nid;
+            BitLength = bitLength;
+            GNBValue = gnbValue;
+            MccInfoConnectionInfoGlobalRanNodeIdPlmnIdMcc = mccInfoConnectionInfoGlobalRanNodeIdPlmnIdMcc;
+            MncInfoConnectionInfoGlobalRanNodeIdPlmnIdMnc = mncInfoConnectionInfoGlobalRanNodeIdPlmnIdMnc;
+            LocationType = locationType;
+            Tac = tac;
+            MccInfoConnectionInfoLocationInfoPlmnMcc = mccInfoConnectionInfoLocationInfoPlmnMcc;
+            MncInfoConnectionInfoLocationInfoPlmnMnc = mncInfoConnectionInfoLocationInfoPlmnMnc;
+            FivegTmsi = fivegTmsi;
+            RegionId = regionId;
+            SetId = setId;
+            Pointer = pointer;
+            MccInfoFivegGutiPlmnMcc = mccInfoFivegGutiPlmnMcc;
+            MncInfoFivegGutiPlmnMnc = mncInfoFivegGutiPlmnMnc;
+            RatType = ratType;
+        }
+
+        /// <summary> Initializes a new instance of <see cref="UeInfo5G"/> for deserialization. </summary>
+        internal UeInfo5G()
+        {
+        }
+
+        /// <summary> Subscription Permanent Identifier. </summary>
+        public string Supi { get; set; }
+        /// <summary> Permanent Equipment Identifier. </summary>
+        public string Pei { get; set; }
+        /// <summary> Gets the session info. </summary>
+        public IList<UeSessionInfo5G> SessionInfo { get; }
+        /// <summary> Per-UE transport network layer association. </summary>
+        public string PerUeTnla { get; set; }
+        /// <summary> The AMF UE NGAP ID. </summary>
+        public long? AmfUeNgapId { get; set; }
```

Comment:
same question for the format of the ID

Reply:
Yes. It must be long. But I am not sure why  "dotnet build /t:GenerateCode" generated it as nullable long.